www/src/components/guidelines/containers.js

```diff
@@ -28,12 +27,8 @@ export const Section = ({ children, ...rest }) => (
     sx={{
       ...rest,
       ...container,
-      py: 4,
-      [mediaQueries.sm]: {
-        py: 5,
-      },
+      py: [4, null, 5, 8],
       [mediaQueries.md]: {
-        py: 8,
         px: 10,
```

Comment:
Should we do `px: [null, null, null, 10]` here so that the related padding values are a bit easier to understand?

Reply:
Yeah, I thought that would be an overkill, but I will take care of it from next time.